data_model/src/peer.rs

```diff
@@ -27,19 +26,15 @@ pub mod model {
     ///
     /// Equality is tested by `public_key` field only.
     /// Each peer should have a unique public key.
-    #[derive(
-        Debug, Display, Clone, Eq, Getters, Decode, Encode, Deserialize, Serialize, IntoSchema,
-    )]
+    #[derive(Debug, Display, Clone, Eq, Decode, Encode, Deserialize, Serialize, IntoSchema)]
     #[display(fmt = "{public_key}@@{address}")]
-    #[getset(get = "pub")]
     #[ffi_type]
     pub struct PeerId {
         /// Address of the [`Peer`]'s entrypoint.
         // TODO: Derive with getset once FFI impl is fixed
-        #[getset(skip)]
         pub address: SocketAddr,
         /// Public Key of the [`Peer`].
-        pub public_key: PublicKey,
+        pub public_key: Box<PublicKey>,
```

Comment:
Without it `PeerId` takes a lot of space so that clippy starts complaining about `RegisterBox` enum.

I think fixing that warging on `RegisterBox` side would be kind of inconsistent with other instructions

Reply:
ok I guess